.github/workflows/generate-template.yaml

```
@@ -5,11 +5,11 @@ on: [push]
 permissions:
   contents: write
 
-# if condition must be changed to github.repository = 'solid-software/flutter_project_template'
+# if condition must be changed to github.repository != 'solid-software/flutter_project_template'
 
 jobs:
   create-project:
-    if: ${{ github.repository	!= 'solid-software/flutter_project_template' }}
+    if: ${{ github.repository	== 'solid-software/flutter_project_template' }}
```

This isn't right. The users would have to manually trigger the pipeline, which isn't ideal.

The whole purpose of this line is to disable this pipeline in the template repo.
Whenever a clone happens, the repo name changes and triggers the generation script.
```suggestion
    if: ${{ github.repository != 'solid-software/flutter_project_template' }}
```

---

You are right.
It must be changed before merge to master. I changed this line in my repo, for avoiding generating template in own repo